truss/cli/cli.py

```diff
@@ -409,13 +409,22 @@ def predict(
     default=False,
     help="Trust truss with hosted secrets.",
 )
+@click.option(
+    "--promote",
+    type=bool,
+    is_flag=True,
+    required=False,
+    default=False,
+    help="After deploy completes, promotes the truss to production.",
```

Comment:
What happens if we use `--promote` without `--publish`? Does it create a development deployment and promote from development to production?

Reply:
good question -- I think if you do --promote without --publish, it should assume `--publish`.

Reply:
i'll make that change